### ISSUE TITLE
중간 평가 피드백 (이 이슈는 closed 하지 말아요.)

### ISSUE THREAD

**Issue by yuhanappdev** [rhtjsdud1986/AllInOneNote-AION-#5]
먼저 Github을 아마 처음 사용해봤을 건데 익숙치 않은 툴을 사용하느라 고생많았어요.

피드백 하기 전에 강의에서 얘기했던 칸반보드와 기획서의 용도에 대해서 다시 언급하면 칸반보드는 작성하는 시점에서 향후에 해야될 모든 일들을 계획하면서 전체 일정과 범위를 계산하는 것에 큰 의미가 있어요. 전체를 알아야 부분을 언제까지 해야되고 이 계획이 잘 될지 안될지 가늠할 수 있습니다. ​

기획서는 나 자신 그리고 제3자와 소통하기 위한 수단입니다. 즉, 최대한 모든 사항이 적혀 있어야 해당 사항에 대해서 3자 혹은 나 자신이 생각하는 것을 명확하게 이해할 수 있어요. 특히 현업에서는 기획리뷰를 모든 유관부서와 함께 하는데 모든 사항이 적혀있지 않다면 기획 리뷰 시 많은 의문과 대답으로 필요없는 시간을 버릴 수 있어요. ​

프로젝트에 이슈가 연동이 안되어 있는데 이렇다고 한다면 마일스톤으로 관리할 수 없기에 계획이라는 측면에서 어느 일정으로 잘 진행되어 가고 있는지 확인할 수 없어요. 
그리고 항목들이 얼핏 세부적으로 잘 나누어져 있는데 해당 항목들이 진짜로 관리되어야 하는 항목인지는 의문이 생기네요. 
예를 들어서 완료된 항목 중에 스트롤바 기능 기획이라는 것이 있는데 완료된 항목인데 기획서에 스크롤바에 관련된 내용은 없어 보이네요. 그리고 스크롤바라는 것이 기능이 정해져 있는 항목인데 그 기획이라고 한다면 명확하게 스크롤바를 재정의해서 어떻게 사용한다라는 것이 있어야 할 것 같은데 역시 눈에 보이지 않아요. 다른 완료된 기능항목도 전부 마찬가지입니다. 

기획서도 현재 상태에서는 기획서 혹은 사업계획서 혹은 디자인 시안 그 어딘가에 걸쳐져 있어 보이네요. 기획서라고 한다면 명확한 기능 정의와 해당 기능이 동작하는 방식, 그리고 화면에 어떻게 나올지가 모두 있어야 되는데 현재 기획서에서 명확한 부분은 없어요. 
앞에 얘기한 연장선상에서 일기장, 가계부, 스터디플래너, 달력이 모두 포함된 앱이라고 되어 있는데 실제 동작화면에서는 그 분류를 어떻게 구분하는지 나와있지 않아요. 
그리고 보통 노트 앱들은 노트북을 먼저 선택하고 그 안에서 노트를 추가하는 방식인데 일반적인 방식을 따라가지 않는다면 어떻게 동작할지 명확하게 기재해주어야 되요. 

전체적으로 프로젝트 계획, 기획서 모두 미흡해보입니다. 해당 내용으로 과연 앱을 개발할 수 있을지 걱정이 됩니다. 실제 프로그래밍 작업에 들어가기 전에 어떤 작업을 할 것인지 명확하게 정의하면 좋겠어요.

여튼간에 작성하느라 고생많았습니다. 기말에는 좀더 나아지길 기대할께요.

**Comment by rhtjsdud1986** [comment]
안녕하세요 교수님 우선 단호하고 상세한 피드백 정말 감사드립니다ㅠㅠ!!! 

틀이 정해져있지 않아 이것저것 넣다보니 이도저도 아닌 기획서가 되어버린것 같아 저도 많이 아쉽게 제출했습니다. 이렇게 하는 게 맞는지 의문도 들고요ㅜㅜ 덕분에 다음부터 기획서는 어떻게 써야할지 알게되어 좀 더 발전시킬 수 있을 것 같습니다!!

중간평가는 끝나버렸지만 기획서도 칸반보드도 더 보완하여 제대로 앱을 마무리지을 수 있도록 열심히 하겠습니다!

**Comment by yuhanappdev** [comment]
예 :) 항상 얘기하지만 모든 것은 정답이 아닌 최선을 찾아가는 고된 과정이에요 고선영학생도 점차 방법을 잘 찾아가리라 생각합니다. 

학기 끝까지 화이팅해요.
